packages/edit-site/src/components/global-styles/variations/variations-color.js

```diff
@@ -24,9 +24,13 @@ export default function ColorVariations( { title, gap = 2 } ) {
 			{ title && <Subtitle level={ 3 }>{ title }</Subtitle> }
 			<VStack spacing={ gap }>
 				{ colorVariations.map( ( variation, index ) => (
-					<Variation key={ index } variation={ variation } isPill>
-						{ () => <StylesPreviewColors /> }
-					</Variation>
+					<Tooltip key={ index } text={ variation?.title }>
+						<div>
```

Comment:
Do we need this extra div?

Reply:
Hi @scruffian, thanks for your review.

I tried it without the div, but it seems that the Tooltip doesn't seem to work then.